### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by thoughtassassin** [paliarush/vagrant-magento#228]
Running `bash init_project.sh` fails. I see this message in the log: 

`[2019-06-10 20:05:18] STATUS: >  ./bin/magento setup:install     --db-host=localhost     --db-name=magento     --db-user=root     --backend-frontname=admin     --base-url=http://magento2.vagrant148/     --language=en_US     --timezone=America/Chicago     --currency=USD     --admin-lastname=Admin     --admin-firstname=Admin     --admin-email=admin@example.com     --admin-user=admin     --admin-password=123123q     --cleanup-database     --use-rewrites=1     --amqp-host=localhost     --amqp-port=5672     --amqp-user=guest     --amqp-virtualhost=/     --amqp-password=guest [/vagrant/scripts/guest/m-reinstall]]`

`Magento supports PHP 7.1.3 or later. Please read https://devdocs.magento.com/guides/v2.3/install-gde/system-requirements-tech.html`

`[2019-06-10 20:05:18] ERROR: >  Magento installation failed. [/vagrant/scripts/guest/m-reinstall]]`

When I `vagrant ssh` and then `php --version`, I see this version:

PHP 7.0.7-4+deb.sury.org~trusty+1

When I tried to update my config.yaml to `php_version: "7.2"` and `vagrant reload` I see two messages regarding PHP:

`
 update-alternatives: error: 
 alternative /usr/bin/php7.2 for php not registered; not setting
 ERROR: Module php7.1 does not exist! [Unknown source file]
 ERROR: Module php7.2 does not exist! [Unknown source file]
`

And when I ssh into the box and look at the contents of /usr/bin/:

![Screen Shot 2019-06-10 at 3 29 53 PM](https://user-images.githubusercontent.com/1539057/59224803-da930680-8b94-11e9-885f-0e5ae6fa00ea.png)

**Comment by JackWormUK** [comment]
Yes UBUNTU 14 is EOL and the PHP packages are no longer available to install.  Until @paliarush updates the Ubuntu version this project no longer works.  There is nothing you can do to fix the error.

**Comment by mavinbe** [comment]
@thoughtassassin
I've made a fork wich runs wich uses a debian/buster64 vagrant box. 
I made many changes to make it run.
It's not well tested but you can have a try:
https://github.com/mavinbeh/magento2-vagrant-for-developers
